deploy/helm/magda-dev.yml

```diff
@@ -98,6 +98,9 @@ correspondence-api:
   smtpHostname: "smtp.mailgun.org"
   smtpPort: 2525
 
+minion-broken-link:
+  domainWaitTimeConfig: '{"data.csiro.au":60,"data.gov.au":60,"data.act.gov.au":30}'
```

Comment:
You can actually have these specified as YAML then dynamically JSONify them - see gateway/templates/configmap.yaml :)

Reply:
Yeah~ that will make config file much nicer 😄 
Have updated~ Please have a look~